src/app/actions.js

```diff
@@ -0,0 +1,8 @@
+export const RESET_STATE = "RESET_STATE";
```

Comment:
This is a hack used to recover the persisted state and also used from our test to simulate different states